watched_keywords.txt

```diff
@@ -2992,3 +2992,4 @@
 1523612169	Glorfindel	crowdbabble\.com
 1523613620	tripleee	ibrowhenna\.com
 1523617067	Glorfindel	pentalift\.com
+1523619508	Olivia Zoe	pinpinterest\.com
```

Comment:
needs to be `pinpinterest\W?com`: https://chat.stackexchange.com/transcript/message/43985990#43985990